src/scss/_media.scss

```diff
@@ -0,0 +1,46 @@
+$phone-large-width: 25em; //400px
+$tablet-small-width: 40em; //640px
+$tablet-large-width: 47em; //752px
+$desktop-small-width: 60em; //960px;
+$desktop-width: 77em; //1200px
+$desktop-large-width: 95em; //1520px
+
+//Size that grid elements are contained at
+$contain: 75em; //1200px
+$contain-nav: 87.5em; //1400px
+
+@mixin phone-large {
```

Comment:
Basically avoid device naming and size naming when it comes to variables regarding media queries.
Neither are true.

But also the mixins are more work than necessary to create media queries.